packages/moonstone/styles/internal/fonts.less

```diff
@@ -46,15 +46,28 @@
 	pa: local("LG Smart UI Gurmukhi");
 	ta: local("LG Smart UI Tamil");
 	te: local("LG Smart UI Telugu");
-	ur: local("LG Smart UI Urdu");
-	zh-HK: local("LG Smart UI TC");
-	zh-TW: local("LG Smart UI TC"); // same as `zh-HK`
+	ur: local("LG Smart UI Urdu") normal 300 600 700;
```

Comment:
Could you have done:
```
ur: local("LG Smart UI Urdu") normal 300 600 700; local("LG Smart UI_Urdu") normal 300 600 700;
```

Reply:
Yes, that would likely have worked. It would add 8 Mac-specific rules to our TV framework for a font that should only be installed on a TV or one of our framework development machines, in a language that none of us can read. If it's important to be able to notice the difference between standard Arabic LG Smart UI and Urdu LG Smart UI, then it may be valuable to add; however, it seemed like it would be fine to just have the TV version supported at this time. I'm open to other opinions though. I'm hoping that adding rules to fonts that don't exist doesn't mess up existing font-face definitions.